src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -1,9 +1,11 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
-  
+
+
   <PropertyGroup Label="Targets required for unit tests to run">
-    <TargetFrameworkNetDesktop461>net461</TargetFrameworkNetDesktop461>
+
+    <TargetFrameworkNetDesktop461>net462</TargetFrameworkNetDesktop461>
```

Comment:
Can you please revert moving to 4.6.2 if it is not stricly needed? We should do it when we need to only.